gradle-suppressible-error-prone/src/main/java/com/palantir/gradle/suppressibleerrorprone/SuppressibleErrorPronePlugin.java

```diff
@@ -75,15 +77,21 @@ private void applyToJavaProject(Project project) {
                 .orElseThrow(
                         () -> new RuntimeException("SuppressibleErrorPronePlugin implementation version not found"));
 
-        // When auto-suppressing, there are two stages:
-        // 1. The first runs a bytecode patched version of errorprone (via an
-        //    artifact transform) that intercepts every error from every check and adds a custom fix, a
-        //    @RepeatableSuppressWarnings annotation to the relevant statement/method/field/class.
-        // 2. The second stage runs a single errorprone check: SuppressWarningsCoalesce, which will combine all
-        //    the @RepeatableSuppressWarnings and @SuppressWarnings annotations into one normal @SuppressWarnings
-        //    annotation.
-
-        setupErrorProneArtifactTransform(project);
+        // If we're going to remove suppressions, and possibly apply patches, we don't want to apply the custom
+        //   logic for for-rollout suppressions.
+        // Note that this means we need to handle the requested patches with care, so as to not apply patches to
+        //   checks that are suppressed with for-rollout, but for which we're not going to remove the suppressions.
+        if (!isRemovingSuppressions(project)) {
+            // When auto-suppressing, there are two stages:
+            // 1. The first runs a bytecode patched version of errorprone (via an
+            //    artifact transform) that intercepts every error from every check and adds a custom fix, a
+            //    @RepeatableSuppressWarnings annotation to the relevant statement/method/field/class.
+            // 2. The second stage runs a single errorprone check: SuppressWarningsCoalesce, which will combine all
+            //    the @RepeatableSuppressWarnings and @SuppressWarnings annotations into one normal @SuppressWarnings
+            //    annotation.
```

Comment:
I know this is unrelated, but this is actually wrong now, really it's just stage 1 and doesn't produce `@RepeatableSuppressWarnings` - I must have forgotten to update it.

Reply:
Should I remove the entire comment? I don't think I understand enough of the internals to provide a properly updated comment. Proposal would be
```
When auto-suppressing, the logic will run a bytecode patched version of errorprone (via an artifact transform) that intercepts every error from every check and adds a custom fix
```
but I'm not sure that's correct

Reply:
yeah, that's good